components/Logo/LogoLockup.tsx

```diff
@@ -45,7 +45,7 @@ export const LogoLockup = ({
     return (
       <Link
         className={cnb(styles.root, className)}
-        href="/"
+        href="/home"
```

Comment:
Home link is now to the slug. A redirect makes sure the end user doesn't see it. This is to support the RSC prefetching and Safari where the back button broke when RSC prefetching returned a 500 error. 

The Safari bug is also tied to the ODB Function/ISR functionality somehow but I haven't troubleshooted it and am at a 'it works don't touch it' place.